rpc/params/storage.go

```diff
@@ -913,6 +913,11 @@ type ImportStorageParams struct {
 
 	// StorageName is the name of the storage to assign to the entity.
 	StorageName string `json:"storage-name"`
+
+	// Force indicates whether to force the import operation, for example
+	// by deleting an existing PVC that is bound to the PV but not used
+	// by another Juju application.
+	Force bool `json:"force,omitempty"`
```

Comment:
The comment here is specific t k8s - this is a generic struct for any storage type so the comment needs to be generic